ultravox/tools/gradio_demo.py

```diff
@@ -1,42 +1,151 @@
 from dataclasses import dataclass
-from typing import Tuple
+from typing import Optional
 
 import gradio as gr
-import numpy as np
 import simple_parsing
 
 from ultravox.data import datasets
 from ultravox.inference import ultravox_infer
 
+demo_instruction: str = """Enter your prompt here (audio will be inserted at the end or at <|audio|>).
+
+Text mode: Shift+Enter to submit.
+Voice mode: Click the recording button to start, then click again to stop and submit.
+"""
+
 
 @dataclass
 class DemoConfig:
     # model_path can refer to a HF hub model_id, a local path, or a Weights & Biases artifact
     #    fixie-ai/ultravox
     #    runs/llama2_asr_gigaspeech/checkpoint-1000/
     #    wandb://fixie/ultravox/model-llama2_asr_gigaspeech:v0
-    model_path: str = "fixie-ai/ultravox"
-    default_prompt: str = "Transcribe\n<|audio|>"
+    model_path: str = "fixie-ai/ultravox-v0_3"
+    # "mps", "cuda", or "cpu"
+    device: str = "mps"
```

Comment:
These values (`device` and `data_type`) should probably be omitted because `UltravoxInference` will auto-select the best options automatically.

Reply:
I will update `ultravox.inference.utils.default_device()` to include `mps`. But it is still good to keep the options here (default `None`), e.g., testing `cpu` speed on Macbook.